packages/big-design/src/components/TableNext/types.ts

```diff
@@ -44,13 +49,15 @@ export interface TableProps<T> extends React.TableHTMLAttributes<HTMLTableElemen
   actions?: React.ReactNode;
   columns: Array<TableColumn<T>>;
   emptyComponent?: React.ReactElement;
+  expandedRowSelector?: (item: T) => T[] | undefined;
```

Comment:
I think should be better to put `expandedRowSelector` inside `TableExpandable` 🤔